Vagrantfile

```diff
@@ -152,53 +152,52 @@ if ! grep -Fxq "$(cat .ssh/#{PRESHARED_SSH_KEY_NAME}.pub)" .ssh/authorized_keys
 fi
 SCRIPT
 
+INSTALL_PYTHON = <<-SCRIPT
+  #!/bin/bash
```

Comment:
Why this indentation?

Reply:
Fixed by addressing @slaperche-scality 's comments.